hasher-matcher-actioner/hmalib/common/configs/actioner.py

```diff
@@ -40,6 +40,14 @@ def perform_action(self, match_message: MatchMessage) -> None:
         raise NotImplementedError
 
 
+# at match time, these strings are replaced by the result of running the associated
+# function on the match message
+WEBHOOK_ACTION_PERFORMER_REPLACEMENTS: t.Dict[str, t.Callable[[MatchMessage], str]] = {
+    "<content-id>": lambda match_message: match_message.content_key,
+    "<content-hash>": lambda match_message: match_message.content_hash,
```

Comment:
I don't see how `<content-hash>` is useful, especially where the usual expectation is that it will be HMA generated

Reply:
Same comment as on headers. I don't think its common but don't see any downside to including it

Reply:
The downside imo is the added complexity and the possibility of it breaking something unexpected